xcm-simulator/Cargo.toml

```diff
@@ -29,16 +29,18 @@ polkadot-parachain = { git = "https://github.com/paritytech/polkadot", branch =
 pallet-assets = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.37" }
 
 # ORML Pallets
-orml-xtokens = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37"}
-orml-traits = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37"}
-orml-xcm-support = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", default-features = false, branch = "polkadot-v0.9.37"}
+orml-xtokens = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git", branch = "polkadot-v0.9.37"}
+orml-traits = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git",  branch = "polkadot-v0.9.37"}
+orml-xcm-support = {git = "https://github.com/open-web3-stack/open-runtime-module-library.git",  branch = "polkadot-v0.9.37"}
 
 # Custom pallets
 xcm-helper = {path = "../pallets/xcm-helper" }
 asset-handler = {path = "../pallets/asset-handler"}
-router = {path = "../pallets/router"}
-pallet-amm = {path = "../pallets/swap"}
-support = {path = "../pallets/support"}
+pallet-amm = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = "gj-parachain" }
+support = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = "gj-parachain" }
+router = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = "gj-parachain" }
+thea-message-handler = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = "gj-parachain" }
+thea-primitives = { git = "https://github.com/Polkadex-Substrate/Polkadex.git", branch = "gj-parachain" }
```

Comment:
Same

Reply:
Same as [here](https://github.com/Polkadex-Substrate/parachain/pull/35#discussion_r1187507349).

Reply:
Added issue #43 to fix this when pointing branch will be merged. 
Resolving this comment.